src/test/java/com/ncc/aif/NistTA3ExamplesAndValidationTest.java

```diff
@@ -154,14 +163,17 @@ void valid() {
 
             @BeforeEach
             void setup() {
-                ImmutablePair<Resource, Resource> relationPair = utils.makeValidNistRelation(SeedlingOntology.GeneralAffiliation_APORA);
+                ImmutablePair<Resource, Resource> relationPair = utils.makeValidNistRelation(
+                        LDCOntology.GeneralAffiliation_ArtifactPoliticalOrganizationReligiousAffiliation);
                 relation = relationPair.getKey();
                 relationCluster = relationPair.getValue();
                 eventCluster = makeClusterWithPrototype(model, utils.getClusterUri(), event, system);
 
                 // This isn't strictly needed to be valid, but it's here because the example looks incomplete if an
                 // entity has a relationship without a relation edge defining that relationship.
-                relationEdge = utils.makeValidTA3Edge(relation, SeedlingOntology.GeneralAffiliation_APORA_Affiliate, entity, 102.0);
+                relationEdge = utils.makeValidTA3Edge(relation,
+                        LDCOntology.GeneralAffiliation_ArtifactPoliticalOrganizationReligiousAffiliation_Artifact,
```

Comment:
Should probably be GeneralAffiliation.ArtifactPoliticalOrganizationReligiousAffiliation_EntityOrFiller as entity is a PER. Only including this comment once, but it applies in other places.

Reply:
Fixed